.github/workflows/build.yml

```
@@ -28,24 +28,35 @@ jobs:
           find Std -name "*.lean" | LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > Std.lean
 
       - name: build std
+        id: build
         run: lake build
 
       - name: test std
+        if: steps.build.outcome == 'success'
         run: make test
 
       - name: lint std
+        if: steps.build.outcome == 'success'
         run: make lint
 
       - name: check that all files are imported
+        if: always()
         run: git diff --exit-code
 
       - name: Check for forbidden character ↦
+        if: always()
         run: |
           if grep -r --include=\*.lean -e '↦' . ; then
             echo "Error: Found forbidden character ↦"
             exit 1
           fi
 
       - name: Check for long lines
+        if: always()
         run: |
           ! (find . -name "*.lean" -type f -exec grep -E -H -n '^.{101,}$' {} \; | grep -v -E 'https?://')
+
+      - name: Check for forbidden 'import Lean'
```

```suggestion
      - name: Don't 'import Lean', use precise imports
```
Hopefully this should help clarify why this lint exists